components/brave_wallet/browser/solana_transaction.cc

```diff
@@ -212,6 +212,43 @@ SolanaTransaction::GetSignedTransactionBytes(
   return transaction_bytes;
 }
 
+// Get serialized and signed transaction given a signature from a hardware
+// wallet. Only supports transactions with one signer.
+absl::optional<std::vector<uint8_t>>
+SolanaTransaction::GetSignedTransactionBytes(
+    const std::vector<uint8_t>& signature_bytes) const {
+  if (signature_bytes.size() > kSolanaSignatureSize)
```

Comment:
It needs to be the exact size, should be `== kSolanaSignatureSize` here.

Reply:
It looks like this is an error case check. Would we want this to be `!= kSolanaSignatureSize` instead?

Reply:
Yea sorry I wasn't clear, I meant the valid case needs to be exact. So yes for error check it's !=.

Reply:
Thanks, just wanted to make sure I wasn't misunderstanding something 😄